cli/network.go

```diff
@@ -125,3 +126,61 @@ func networkCreateExample() string {
 	return `$ pouch network create -n pouchnet -d bridge --gateway 192.168.1.1 --ip-range 192.168.1.1/24 --subnet 192.168.1.1/24
 pouchnet: e1d541722d68dc5d133cca9e7bd8fd9338603e1763096c8e853522b60d11f7b9`
 }
+
+// networkRemoveDescription is used to describe network remove command in detail and auto generate command doc.
+var networkRemoveDescription = "Remove a network in pouchd. " +
+	"It must specify network's name."
+
+// NetworkRemoveCommand is used to implement 'network remove' command.
+type NetworkRemoveCommand struct {
+	baseCommand
+
+	name       string
+}
+
+// Init initializes NetworkRemoveCommand command.
+func (n *NetworkRemoveCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "remove [OPTIONS] [NAME]",
```

Comment:
If `cobra.ExactArgs(1)`, then Use should be `remove [OPTIONS] NAME`

In addition, it is `pouch network rm` or `pouch network remove`?